Remotion/ObjectBinding/Web/Res/HTML/BocAutoCompleteReferenceValue.jquery.js

```diff
@@ -1045,9 +1045,8 @@
                 activeItem.addClass (CLASSES.ACTIVE);
                 activeItem.attr("aria-selected", "true");
 
-                select(false);
                 // TODO provide option to avoid setting focus again after selection? useful for cleanup-on-focus
```

Comment:
The comment existed before the initial change I made, I thought this would result in a separate issue where the focus setting could be disabled at this specific point.

Reply:
Yeah, the comment meant that I didn't like it that there was an input.focus() there when though we already did a select() which could/should have set the focus in some cases. Now, the comment's redundant/solved. 

Yes, it's possible there was another idea hidden in there, too, but the comment isn't clear and wouldn't be helpful to future changes in this part of the code.